packages/core/src/Modifiers.ts

```diff
@@ -247,6 +247,22 @@ export class Modifiers extends EventMixin {
     toggle(modifier: Modifier | Constructor<Modifier>): void {
         this.remove(modifier) || this.append(modifier);
     }
+    /**
+     * Check that all `otherModifiers` ar contained within this ones.
+     *
+     * @param otherModifiers
+     */
+    contains(otherModifiers: Modifiers): boolean {
+        for (const otherModifier of otherModifiers) {
+            const foundModifier = this.find(m => m.isSameAs(otherModifier));
+            // Modifier.isSameAs(undefined) could return true. See
+            // `Attribute.isSameAs`.
```

Comment:
`Attribute` est un plugin non ? Je ne le mentionnerais pas dans core. Tu peux peut-être plutôt expliquer pourquoi on peut se retrouver dans cette situation ?

Reply:
It's tricky to explain I have removed Attribute.isSameAs. Even though it does not shock me to make a reference to another plugin within a comment in core if it help to understand. Do you have an explanation that comes to your mind?

Reply:
"Modifier.isSameAs(undefined) may return true in the case of modifiers that are themselves containers of values. Then we might want to equate the presence of an "empty" modifier with the absence of that modifier."

Reply:
I find it accurate but difficult to understand for someone not familiar. Moreover, technically a modifier that does not represent a container can return true if undefined is provided. 

The example of `Attribute.isSameAs` would give me more information IMO. That's no t a problem to have an external link to explain or provide an example to a concept.

I've changed for your comment anyway.

Reply:
Fair enough. I'd do both then.